web_sdk/BUILD.gn

```diff
@@ -116,7 +116,6 @@ prebuilt_dart_action("flutter_dartdevc_kernel_sdk_outline") {
   script = "//third_party/dart/utils/bazel/kernel_worker.dart"
 
   args = [
```

Comment:
I think it would be helpful to see an explicit `--no-sound-null-safety` on these actions that are labeled as being unsound.

Reply:
Should I make these changes, are or we good without them?

Reply:
Up to you, I'd like to see them there but I don't really maintain this code. If you want to follow up with a change that adds them I'd be happy too. 😄

Reply:
The thing I'm still not clear on is why we'd label these as unsound - is the idea that unsound is the superset of sound and unsound?

Alternatively - where do we build the sound web_ui_library?

Reply:
Ohh I see the comment now, and below where we specify --sound-null-safety.. Ok this makes sense to me now.

Reply:
Added the suggested flags. Thanks!